docs/static/releasenotes.asciidoc

```diff
@@ -11,6 +11,16 @@ See also:
 * <<release-notes-xls>>
 endif::include-xpack[]
 
+[[logstash-6-0-1]]
+=== Logstash 6.0.1 Release Notes
+* Elasticsearch Output: Ignore event's type field for the purpose of setting document _type if cluster is es 6.x or above
+* Fix bug where plugins that used nested hashes for configuration would not work correctly
+* Ensure logstash always terminates on second sigint, there were some situations where executing two sigints would not kill Logstash
+* Fix bug where persistent queue could not be opened when in some states
+* Fix an issue where the aggregate filter would not always shut down correctly
```

Comment:
Incidentally, not sure if it makes sense for these particular issues, but we used to provide more links to related issues in the release notes (see the relnotes for the 5.x releases). We would include a link parenthetically at the end of a sentence. The asciidoc coding looks like this (but only for issues in the logstash repo, or course):

...blah blah blah ({lsissue}1111[Issue 1111]).